spec/views/hyrax/base/_form_metadata.html.erb_spec.rb

```diff
@@ -134,8 +134,8 @@
     end
 
     it 'has licenses' do
-      expect(page).to have_form_field(:license)
-        .as_single_valued
+      expect(page)
+        .to have_multivalued_field(:license)
```

Comment:
`#have_multivalued_field` is deprecated. Fields are now assumed multi-valued unless `.as_single_valued` is specified.